Maestro/subscriptions.json

```diff
@@ -6,6 +6,13 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 1084
     },
+    // A build definition capable of running any .ps1 script in the CoreFX repo dev/api branch
+    "corefx-general-dev-api": {
+      "vsoInstance": "devdiv.visualstudio.com",
+      "vsoProject": "DevDiv",
+      "vsoSourceBranch": "dev/api",
```

Comment:
@eerhardt Did I set this properly? Great to be able to do this instead of cloning the def again. :+1: